R/viz.R

```diff
@@ -3710,6 +3716,28 @@ fancy_ideogram = function(this_sample,
     cn_states = mutate(cn_states, chrom = paste0("chr", chrom))
   }
 
+  if(intersect_regions){
+    #filter CN states on intersecting regions
+    #transform regions to Granges objects
+    grl = GRanges(cn_states)
```

Comment:
Alright, see it here now! Can we consider switching this to `data.table::foverlaps()`?

Reply:
Yes, I will do this. Good suggestion!